src/components/resource-section/resource-panel/ResourcePanel.js

```diff
@@ -2,7 +2,7 @@ import { Panel } from 'primereact/panel';
 import { useRouter } from 'next/router';
 import { Button } from 'primereact/button';
 import { Card } from 'primereact/card';
-import LinkButton from '../link-button/LinkButton';
+import LinkButton from '../../common/link-button/LinkButton';
```

Comment:
please, use `@components` alias

Reply:
You are right, it was the automatic change of the linter